tesseract_task_composer/planning/src/nodes/continuous_contact_check_task.cpp

```diff
@@ -160,7 +160,18 @@ ContinuousContactCheckTask::runImpl(TaskComposerContext& context, OptionalTaskCo
   if (contactCheckProgram(contacts, *manager, *state_solver, ci, cur_composite_profile->config))
   {
     info->status_code = 0;
-    info->status_message = "Results are not contact free for process input: " + ci.getDescription();
+    std::stringstream ss;
+    ss << "Results are not contact free for process input: " << ci.getDescription() << "\n";
+    // Iterate over each contact and add collision summary if not empty
+    for (std::size_t i = 0; i < contacts.size(); ++i)
+    {
+      const auto& contact_map = contacts[i];
+      if (!contact_map.empty())
+      {
+        ss << contact_map.getSummary() << "\n";
+      }
+    }
```

Comment:
I could see this blowing up. Should we just print the first contact summary?

Reply:
The way I have it, it's up to the user how much is printed. If they have contact check set to FIRST then they'll only get one. If it is set to ALL then they'll potentially get a long novel printed out, which is maybe useful for them.

Reply:
I think it may be better to compile the vector of maps into a single map and then print the summary to avoid a large message.